firmware/boron-ins-fsm/src/stateMachine.h

```diff
@@ -21,11 +21,14 @@
 
 //length of time between debug publishes
 #define DEBUG_PUBLISH_INTERVAL 1500  //ms
-#define SM_HEARTBEAT_INTERVAL 600000  //ms = 10 min
+#define SM_HEARTBEAT_INTERVAL 660000  //ms = 11 min
```

Comment:
Hmmm, just wondering how all these changes will interact with your OTA disabled/enable code. What do you think?

Reply:
Thats a good question, I think the doorHeartbeatReceived time should still be updated when we only receive a door heartbeat. But I'll use a different variable doorLastMessage time to be updated and published to the particle console under the "name" doorHeartbeatReceived time. 
As for those  variables above, its the IM21 heartbeat_interval that we care about whereas those are editing the boron's heartbeat_interval